tools/sdc-docker-setup.sh

```diff
@@ -109,6 +109,21 @@ function dockerInfo
 }
 
 
+# Return ssh fingerprint in the form "he:xh:ex:he:xh..."
+# OpenSSH_6.8 changes the -l flag output.
+function sshGetMD5Fingerprint() {
+    local s
+    sshPubKeyPath=$1
+    s=$(ssh-keygen -E md5 -l -f "$sshPubKeyPath" 2> /dev/null)
+    if [[ $? -eq  0 ]]; then
+        echo "$s" | awk '{print $2}' | tr -d '\n' | cut -c 5-;
```

Comment:
is `| cut -c 5-;` accurate? Doesn't it use a ':' delimiter?  If so, would prefer `cut -d: -f2-`. Fine either way tho. I'm being pedantic. :)